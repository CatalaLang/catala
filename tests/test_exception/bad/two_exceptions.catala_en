## Test

```catala
declaration scope A:
  output x content integer

scope A:
  label base_x
  definition x equals 0

  exception base_x
  definition x equals 1

  exception base_x
  definition x equals 2
```

```catala-test-inline
$ catala Interpret -s A
[ERROR]
There is a conflict between multiple valid consequences for assigning the same variable.

This consequence has a valid justification:
┌─⯈ tests/test_exception/bad/two_exceptions.catala_en:12.23-12.24:
└──┐
12 │   definition x equals 1
   │                       ‾
   └─ Test

This consequence has a valid justification:
┌─⯈ tests/test_exception/bad/two_exceptions.catala_en:15.23-15.24:
└──┐
15 │   definition x equals 2
   │                       ‾
   └─ Test
#return code 123#
```
