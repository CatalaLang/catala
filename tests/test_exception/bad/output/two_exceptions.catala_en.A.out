[ERROR] There is a conflict between multiple exceptions for assigning the same variable.
[ERROR] 
[ERROR] This justification is true:
[ERROR]   --> test_exception/bad/two_exceptions.catala_en
[ERROR]    | 
[ERROR] 15 |   definition x equals 2
[ERROR]    |              ^
[ERROR]    + Test
[ERROR] 
[ERROR] This justification is true:
[ERROR]   --> test_exception/bad/two_exceptions.catala_en
[ERROR]    | 
[ERROR] 12 |   definition x equals 1
[ERROR]    |              ^
[ERROR]    + Test
